ntbs-service/DataAccess/UserRepository.cs

```diff
@@ -63,9 +63,20 @@ private async Task UpdateUser(User existingUser, User newUser, IEnumerable<TBSer
         private static void AddCaseManagerTbServices(User user, IEnumerable<TBService> tbServices)
         {
             var caseManagerTbServices = tbServices
-                .Select(tb => new CaseManagerTbService {TbService = tb, CaseManager = user})
+                .Select(tb => new CaseManagerTbService
+                {
+                    TbService = tb,

```

Comment:
We don't need to project the navigation properties here, TbServiceCode and CaseManagerUsername should be sufficient.